### ISSUE TITLE
Custom script 

### ISSUE THREAD

**Issue by ernstwys** [unifiedremote/Remotes#132]
Since new updates on my RPI, all my custom script stopped working.

I made some direct link to .sh script. Like for instance this one:

All help is welcome. Been looking into it, but can't find why suddenly these functions are responding,

--@help Command 1
actions.command1 = function ()
        script.default("sh funx.sh");
end

**Comment by pqvst** [comment]
Here are some tips:
- Have you checked the error message in the server log?
- Try `script.default("/bin/sh funx.sh")` instead?
- Try `os.start()` or `os.open()` instead?

https://github.com/unifiedremote/Docs/blob/master/libs/os.md

**Comment by ernstwys** [comment]
Yes, no errors.. i checked the .sh files nothing wrong with that.. also tried your advice.. did not helped. 

Looks like function is not run. Really need it to run the .sh files. No need for os.start() functions
